docker-compose.yaml

```diff
@@ -141,16 +142,34 @@ services:
     depends_on:
       - keycloak
 
-  keycloak: # https://hub.docker.com/r/jboss/keycloak
+  postgres: # https://hub.docker.com/_/postgres
```

Comment:
To keep it consistent with the other definitions, could you also add a `restart` and `networks` property? I assume that postgres only has to be accessible in the `identity-network`.

Reply:
Can you please add a volume definition for the `postgres` container, similar to elastic and zeebe services.

Reply:
Can you also please add a health check to the postgres container, maybe something like
```yaml
healthcheck:
  test: ["CMD-SHELL", "pg_isready"]
  interval: 10s
  timeout: 5s
  retries: 5
```